core/src/toga/__init__.py

```diff
@@ -45,13 +44,15 @@
     "toga.window": ["Window"],
 }
 
+__all__ = []
 
-def __getattr__(name):
 
+def __getattr__(name):
     for module, names in toga_core_imports.items():
         if name in names:
             module = importlib.import_module(module)
             globals()[name] = getattr(module, name)
+            __all__.append(name)
```

Comment:
`__all__` must be generated at the top level of the module, based on the contents of `toga_core_imports`. This is because when the user runs `from toga import *`, the import system needs to know what names to import before it can call `__getattr__`.

Reply:
i believe i do this with commit 176e854, correct me if it's wrong